@navikt/aksel-stylelint/src/deprecations.ts

```diff
@@ -38,4 +38,17 @@ export const deprecations: DeprecatedList = [
     classes: ["navds-modal__content"],
     message: "Renamed to `.navds-modal__body` in v5.0.0",
   },
+  {
+    classes: ["navds-guide-panel--poster"],
```

Comment:
[Ser ut som denne er brukt litt.](https://github.com/search?type=code&q=org%3Anavikt+navds-guide-panel--poster). Litt kjedelig å lage en major på det, kanskje vi kan teste å lage et issue/pr i de repoene for å patche det for dem?

Reply:
Evt gjøre endringen uten å oppdatere klassenavn 🤔

Reply:
God ide!